main.lua

```diff
@@ -40,6 +40,11 @@ function love.update(dt)
 end
 
 function love.draw()
+	local joysticks = love.joystick.getJoysticks()
```

Comment:
`getJoysticks()` is needlessly queried every frame - we should just store this as a variable